assets/css/v2/dup/departures/route_pill.scss

```diff
@@ -35,6 +35,10 @@
     background-color: colors.$line-ferry;
   }
 
+  &--ocean_blue {
+    background-color: colors.$line-capeflyer;
+  }
```

Comment:
We should make corresponding changes to other CSS "implementations" of route pills; this file is only relevant to DUPs.

Reply:
I was intentionally only doing this for DUPs since it's only relevant at South Station and Braintree, but thinking about it more, this does seem safer to future proof for any additional screens we add at these stations.

Reply:
Yeah, I think it'd be best not to leave any unnecessary surprises here for our future selves. The handling of route pills in general is already reasonably "complete" and consistent across various contexts and screen types, so it would be unexpected for this one route to only work on DUPs.